### ISSUE TITLE
Add details to section on new things

### ISSUE THREAD

**Issue by drvinceknight** [Axelrod-Python/An-open-reproducible-framework-for-the-study-of-the-iterated-prisoners-dilemma#10]
In #9 I have added various things to the paper and it's (in very much a draft form that could/should still be ripped a part) a first step. I've put in some very loose details about some of the implications. This can be added to **but at the same time, I'm aware that we might not want to put that much in this paper**, @marcharper could you take a look and A) let me know what you think or B) have a go at it?

Basically, I think it would be great if Marc took a look before I kept going.

**Comment by marcharper** [comment]
I've added some inline comments. Everything looks correct to me otherwise. :+1:
